### ISSUE TITLE
Not work with WebStorm, Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.

### ISSUE THREAD

**Issue by AvailCat** [typescript-eslint/typescript-eslint#1602]
**What code were you trying to parse?**

```ts
// Anything inside my project
```

**What did you expect to happen?**
It should works

**What actually happened?**
It does not work with error:
```
Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: a-folder/some-file.ts.
The file must be included in at least one of the projects provided.
```

**Extra:**
- We have **multiple eslintrc inside different folder**, 
- ESLint was configured to automatic in WebStorm

**Versions**

| package                     | version |
| --------------------------- | ------- |
| `@typescript-eslint/parser` | `2.19.0` |
| `TypeScript`                | `3.7.5` |
| `ESLint`                    | `6.7.2` |
| `node`                      | `12.16.0` |
| `npm`                       | `6.13.4` |

**Comment by armano2** [comment]
can you provide a project where i can reproduce your error, as i'm using webstorm and i do not see issues

**Comment by bradzacher** [comment]
help us to help you, @AvailCat.
The error message by itself is pretty useless.

What does your eslint config look like?
What does your tsconfig look like?
What does the structure of your project look like?
Does this happen on all files or just some?

The more information you put into a bug report, the easier we can help you.

**Comment by kbradl16** [comment]
You get this error banner in webstorm when you do not have the file listed in the `include` section of your tsconfig file that eslint uses.

folder structure:
 - src
      - file.ts
 -index.ts

```
{
  "compilerOptions": {
   ... 
  },
  "include": ["*.ts"], // will report error in all src files
  "exclude": []
}
```

Update it to this
```
{
  "compilerOptions": {
   ... 
  },
  "include": ["*.ts", "src/**/*.ts"], // will lint all .ts files in src
  "exclude": []
}
```

**Comment by bradzacher** [comment]
I'll close this in 3 days if @AvailCat does not respond.
Likely it is due to a misconfiguration.

**Comment by davidgilbertson** [comment]
Thanks @kbradl16! My issue was I have a top level `/src` directory, and `/tests` as a sibling. I'd put `./src` in my `include`, but not `./tests`.
